app/templates/signup.html

```diff
@@ -33,5 +38,7 @@ <h1>Sign up!</h1>
         <p>{{ form.submit2() }}</p>
     </form>
     {% endblock %}
+	</div>
```

Comment:
don't know if this indentation is weird because of how github is displaying it, but if not please fix or it'll bother me.